src/pages/events/[eventId]/edit/index.page.tsx

```diff
@@ -13,11 +16,35 @@ export const getServerSideProps = getApplicationServerSideProps(
   async ({ req, query, queryClient, cookies }) => {
     const { eventId } = query;
 
-    await prefetchGetEventByIdQuery({
-      id: eventId,
-      req,
-      queryClient,
-    });
+    await Promise.all([
+      prefetchGetEventByIdQuery({
+        id: eventId,
+        req,
+        queryClient,
+      }),
+      prefetchGetEventPermissionsQuery({
+        req,
+        queryClient,
+        eventId: eventId,
+      }),
+    ]);
+
+    const permissions = queryClient.getQueryData([
```

Comment:
Can you use desctructuring here?
```suggestion
    const { permissions } = queryClient.getQueryData([
```